src/centos/scripts/start-marklogic.sh

```diff
@@ -12,6 +12,24 @@
 ###############################################################
 
 cd ~
+
+
+################################################################
+# Install Converters if required
+################################################################
+if [[ -z $INSTALL_CONVERTERS ]] || [[ "$INSTALL_CONVERTERS" = "false" ]] ; then
+    echo "Not Installing Converters"
+else
+    if [[ ! -d "/opt/MarkLogic/Converters" ]]; then
```

Comment:
A bit of a nit but since you have a positive and negative condition, I would switch the if/else around and remove the not from the conditional.

Reply:
5f93314710e4ffbfc2370d8dcb77e8d3ef923f49 Updated